tests/v2/validation/extensions/provisioning/permutations/permutations.go

```diff
@@ -0,0 +1,224 @@
+package permutations
```

Comment:
I also think we should move this file directly to v2/validations/provisioning, this function should strictly be used for provisioning testing, so therefore I propose move airgap and registries directly to v2/validation/provisioning as separate test packages.